crates/humanode-rpc/src/lib.rs

```diff
@@ -342,11 +346,14 @@ where
         .into_rpc(),
     )?;
 
+    let eth_tx_pool = TxPool::new(Arc::clone(&client), graph);
```

Comment:
This would look clearer if it the `TxPool` was fully qualified (i.e. with a crate/mod path instead of via `use`).

Reply:
https://github.com/humanode-network/humanode/pull/848/commits/020def58d3d3e4fcc1305ed3f408f57648b88df7